src/index.js

```diff
@@ -41,15 +42,28 @@ export default function ({ Plugin, types: t }) {
     return cls.body.body.filter(isRenderMethod).length > 0;
   }
 
-  const isCreateClassCallExpression = t.buildMatchMemberExpression('React.createClass');
+  function isCreateClassCallExpression(node, factoryMethods) {
+    for (const method of factoryMethods) {
+      if (method.indexOf('.') !== -1) {
+        if (t.buildMatchMemberExpression(method)(node.callee)) {
```

Comment:
It's definitely faster to pre-build it, though I'm not sure how much the difference matters when just about everything is still in the single digit microsecond range:

```
build inline x 2,421,358 ops/sec ±1.20% (91 runs sampled)
prebuilt x 11,400,386 ops/sec ±1.20% (92 runs sampled)
Fastest is prebuilt
build inline x 2,678,951 ops/sec ±1.01% (95 runs sampled)
prebuilt x 11,239,894 ops/sec ±1.51% (90 runs sampled)
Fastest is prebuilt
build inline x 2,634,932 ops/sec ±1.24% (94 runs sampled)
prebuilt x 10,768,092 ops/sec ±1.17% (94 runs sampled)
Fastest is prebuilt
build inline x 296,499 ops/sec ±1.74% (88 runs sampled)
prebuilt x 2,769,726 ops/sec ±0.94% (92 runs sampled)
Fastest is prebuilt
build inline x 1,382,369 ops/sec ±0.97% (93 runs sampled)
prebuilt x 2,686,110 ops/sec ±1.29% (93 runs sampled)
Fastest is prebuilt
build inline x 2,558,054 ops/sec ±1.10% (88 runs sampled)
prebuilt x 11,474,870 ops/sec ±1.19% (93 runs sampled)
Fastest is prebuilt
build inline x 2,553,266 ops/sec ±1.39% (95 runs sampled)
prebuilt x 10,736,413 ops/sec ±1.36% (91 runs sampled)
Fastest is prebuilt
build inline x 1,379,023 ops/sec ±1.25% (93 runs sampled)
prebuilt x 2,820,422 ops/sec ±1.03% (94 runs sampled)
Fastest is prebuilt
```

I'll tweak it to prebuild though, may as well.